### ISSUE TITLE
FSE Beta: Highlight - Color Picker looks broken

### ISSUE THREAD

**Issue by formosattic** [Automattic/wp-calypso#59562]
### Steps to reproduce

1. Add a Headings or Paragraph block
2. Select Highlight option

### What you expected to happen

Have a clear cue of where to click to select a custom color.

### What actually happened

The color picker actually works the display looks strange, just a thin column.
![image1](https://user-images.githubusercontent.com/15944096/147380929-2a369579-c497-4647-bee0-0bcf69e286c3.png)

### Context

Testing FSE Beta on WordPress.com, but also ended up testing on self-hosted and using the FSE plugin at https://wordpress.org/plugins/full-site-editing/ 

Other tests:

* self-hosted site with [FSE plugin](https://wordpress.org/plugins/full-site-editing/): same display happens
* Simple site, self-hosted site with Block editor: also looks off, with checkered area (but that's another topic, see https://github.com/WordPress/gutenberg/issues/36998#issuecomment-983076580 )
![image1](https://user-images.githubusercontent.com/15944096/147381119-ab7d0135-e566-4e9f-b444-0c6007b6a3aa.png)

**Comment by inaikem** [comment]
Note, this issue is related to the WordPress.com Editing Toolkit, so most likely specific to WPcom vs Core.

**Comment by jeyip** [comment]
I ran the latest version of Gutenberg and the [WordPress Editing Toolkit (Full Site Editing) Plugin](https://wordpress.org/plugins/full-site-editing/) on a self-hosted site with the Bennett theme.

I add a paragraph block and select the highlight option, but am unable to reproduce this issue in Chrome, Firefox, Safari, or Edge.  Can any of you still replicate the problem? @inaikem @creativecoder @zaguiini @formosattic

I also no longer see the problem on dotcom simple sites ( I used the Twenty Twenty Two theme ).
